papermill/execute.py

```diff
@@ -106,26 +112,31 @@ def log_outputs(cell):
     for output in cell.get("outputs", []):
         if output.output_type == "stream":
             if output.name == "stdout":
-                stdouts.append("".join(output.text))
+                stdouts.append(u"".join(output.text))
```

Comment:
These shouldn't be necessary anymore.

Reply:
Yeah since we imported Unicode literals from future